subnet/deployment-generator/script/gen_env.js

```diff
@@ -0,0 +1,157 @@
+
+const config = require('./config_gen')
+Object.freeze(config)
+
+module.exports = { 
+  genSubnetConfig,
+  genSubnetConfigMac,
+  genServicesConfig,
+  genServicesConfigMac,
+}
+
+
+
+function genSubnetConfig(subnet_id, key){
+  key_name = `key${subnet_id}`
+  private_key = key[key_name]['PrivateKey']
+  private_key = private_key.slice(2, private_key.length)    //remove 0x for subnet conf
+  port = 20303+subnet_id-1
+  rpcport = 8545+subnet_id-1
+  wsport= 9555+subnet_id-1
+  var config_env = `
+INSTANCE_NAME=subnet${subnet_id}
+PRIVATE_KEY=${private_key}
+BOOTNODES=enode://cc566d1033f21c7eb0eb9f403bb651f3949b5f63b40683917\
```

Comment:
is this bootnode constant

Reply:
I guess so. It doesn't matter what value we use here.

Reply:
I don't get that... so any value will do? Then just put 000

Reply:
I just tested 0000 and aaaaa. It doesn't work. Seems it has to be a correct address format but any valid addr will work.